src/platform/imx8/lib/memory.c

```diff
@@ -10,9 +10,6 @@
 #include <sof/platform.h>
 #include <ipc/topology.h>
 
-STATIC_ASSERT(0 == (HEAP_BUF_ALIGNMENT % PLATFORM_DCACHE_ALIGN),
```

Comment:
Why did you remove this?

Reply:
It doesn't make sense anymore and to be honest it didn't make sense before.

Reply:
@tlauda either mention this in the commit message or make a separate commit. This is totally unrelated with the subject of the commit.

Also, please always explain why a commit is needed in the commit message!